client/my-sites/media/main.jsx

```diff
@@ -72,6 +85,26 @@ class Media extends Component {
 		page( redirect );
 	};
 
+	/**
+	 * Handles updating the current queryFilters
+	 * which are passed into the Media Library
+	 * to allow filtering of Media items.
+	 *
+	 * This method is duplicated in
+	 * `client/post-editor/media-modal/index.jsx` and any changes
+	 * here must also be mirrored across to that location
+	 *
+	 * @param  {object} queryFilters the filters to be updated
+	 */
+	onQueryFiltersChange = queryFilters => {
+		this.setState( {
+			queryFilters: {
+				...this.state.queryFilters,
```

Comment:
My sense is the most common use case is that we will want to append to existing filters rather than nuke existing filters.

For example, if you've filtered by X and Y and then you want to add filter Z you don't want to loose X and Y.

That said in the future we might need a way to completely clear query filters. Perhaps a `clearQueryFilters` method. But it's not a use case at the moment so I'm not inclinded to add it.

Reply:
Sweet, I thought this might be the case :) Thanks Dave!